engine/src/xml/error.rs

```diff
@@ -1,22 +1,23 @@
 use std::fmt::{Debug, Display, Formatter};
 
+// Renaming Error elements to match Rust Clippy suggestion: https://rust-lang.github.io/rust-clippy/master/index.html#/enum_variant_names
 #[derive(Debug)]
 pub enum Error {
-    IoError {},
-    ListModuleError { error: String, path: String },
-    SerdeError {},
+    Io {},
```

Comment:
This is definitely a good convention not to repeat ourselves. I wonder if it makes sense to have an exception to the rule for errors. This is because of
- errors could be imported directly, and code of `Err(Io)` may look a bit weird
- It's easier to grep for `IoError` than it is for `Io`

What do you think?

Reply:
Not really going to be weird. A typical import would be like `Error::Io`, and for bulk, `Error::{Io, Serde}`